docs/pages/application-access/okta/introduction.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Okta integration with Application Access
+description: How to set up Okta integration with Teleport Application Access.
+---
+
+Teleport allows applications housed within Okta to be integrated with the
+Application Access service. This will result in Okta applications showing up
+in Teleport's applications list in the UI and CLI, and it will further allow
+for the user of Teleport's role based access control to control access within
+Okta.
```

Comment:
This is the kind of helpful context that I think is missing in the guide above. _Now_ I understand the value here.

Reply:
TBH I don't know if we need a separate "introduction" page. I would do this:

- Take this paragraph and integrate it at the beginning of the guide above so it's clear what you get from setting up Okta integration.
- Everything below seems more of an "architecture" so I would rename what remains of this page into "Architecture".